### PR TITLE
mesa -> 22.3.1

### PR DESCRIPTION
**Pull request by satmandu**
Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=mesa2231 CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/mesa.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ class Mesa < Package
     @_ver = git_hashtag[0, 7]
     version "amber-#{@_ver}"
   else
-    @_ver = '22.3.0'
+    @_ver = '22.3.1'
```
Comment (uberhacker) [inline packages/mesa.rb:14]: Change to `compatibility 'aarch64,armv7l,x86_64'`?

Comment (satmandu) [reply packages/mesa.rb:14]: There's a version for i686 too though...

Comment (satmandu) [reply packages/mesa.rb:14]: Sorry, left out  the [x] i686 by accident...

Comment (uberhacker) [reply packages/mesa.rb:14]: I don't see the binary here.  Was it missed?

Comment (satmandu) [reply packages/mesa.rb:14]: There is no 22.3.1 binary for i686. 22.3.1 is selected when kernel is >= 5.10, which will never happen on i686.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**